azkaban-common/src/main/java/azkaban/executor/ExecutorUtils.java

```diff
@@ -0,0 +1,62 @@
+package azkaban.executor;
+
+import azkaban.Constants;
+import azkaban.Constants.ConfigurationKeys;
+import azkaban.utils.Pair;
+import azkaban.utils.Props;
+import com.google.common.base.Preconditions;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Executor utility functions */
+public class ExecutorUtils {
```

Comment:
These were factored out to avoid code duplication. If/when ExecutionManager is removed, then these can be moved to ExecutionController. While both classes exist, it's still better to share code, so that fixes or changes will be applied to both code paths.